app/upgrades/v20/upgrades.go

```diff
@@ -40,11 +43,14 @@ func CreateUpgradeHandler(
 		}
 
 		// Initialize the newly created param
-		keepers.ConcentratedLiquidityKeeper.SetParam(ctx, cltypes.KeyUnrestrictedPoolCreatorWhitelist, []string{})
+		keepers.ConcentratedLiquidityKeeper.SetParam(ctx, cltypes.KeyUnrestrictedPoolCreatorWhitelist, emptySlice)
 
 		// Initialize the new params in incentives for group creation.
 		keepers.IncentivesKeeper.SetParam(ctx, incentivestypes.KeyGroupCreationFee, incentivestypes.DefaultGroupCreationFee)
-		keepers.IncentivesKeeper.SetParam(ctx, incentivestypes.KeyCreatorWhitelist, []string{})
+		keepers.IncentivesKeeper.SetParam(ctx, incentivestypes.KeyCreatorWhitelist, emptySlice)
+
+		// Initialize new param in the poolmanager module with a whitelist allowing to bypass taker fees.
+		keepers.PoolManagerKeeper.SetParam(ctx, poolmanagertypes.KeyTakerFeeByPassWhitelist, emptySlice)
```

Comment:
Lets rename this to something like "ReducedTakerFeeWhitelist" so we don't need to change it next upgrade. For now it will just be 100 percent reduced.